src/app/docs/waterfall-indicator/waterfall-indicator-docs.component.html

```diff
@@ -0,0 +1,247 @@
+        <sky-docs-anatomy-item>Completed step action</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Active step action</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Step details </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Start over action </sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Waterfall title"
+      >
+        <p>
+          We strongly recommend including a waterfall title to reflect the context of the overall task. Do not use a left-aligned page title when you include a waterfall title. The waterfall title does not change when users progress through the waterfall steps.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Step details"
+      >
+        <p>
+          You can provide additional details about completed steps, but limit details to single pieces of information. Don’t go overboard because users can reopen modals to review choices as necessary.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Start over action"
+      >
+        <p>
+          You can include a start over action to let users erase all saved data from the waterfall process. This returns user to the first step or to an earlier point in the workflow if decisions occured before the waterfall progress indicator.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior & states"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Waterfall progress indicators guide users through one active step at a time. When users complete a step, the progress indicator updates its step indicator and step action to the completed state and updates the next step indicator and step action to the active state. Users can review and edit completed steps, but this does not undo the completed state.
+        </p>
+        <p>
+          The final step summarizes the changes and requests user confirmation. You do not need to include all data that users enter in the summary, but confirmation is required. After users complete the final step, replace the waterfall progress indicator with a page that displays the finished or active state of the system.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Use a waterfall title in place of an ordinary page title to provide context for the overall task. Begin the title with a verb to describe the action that the waterfall steps take when they are all completed, similar to the way modal titles are constructed.
```

Comment:
It's existing content, but the second sentence is hard to follow. I'd change it to `Follow the same pattern as modal titles and start the waterfall title with a verb that indicates the action that occurs when users complete the waterfall steps.` Or something like that. What do you think?

Reply:
I updated this locally and will push up the change shortly. Resolving the comment ahead of that.